client/client.go

```diff
@@ -87,6 +87,24 @@ func (c *Client) doWithErr(req *http.Request) (*http.Response, error) {
 	return res, nil
 }
 
+func isAlphanumeric(word string) bool {
+	return regexp.MustCompile(`^[a-zA-Z0-9]*$`).MatchString(word)
```

Comment:
It's not very efficient to do this every single time someone calls `isAlphanumeric` - make the regexp a package level variable that gets initialized one time.

Note, I believe this will also pass if `word` is the empty string. It would be pretty easy to add some basic smoke tests for this function.